README.md

```diff
@@ -28,22 +28,22 @@ Please feel free to contribute by forking and submitting pull requests or by
 submitting feature requests or issues to [issues][githubissues].
 
 ## Requires
-* pytest >= 2.2.3
+* pytest >= 2.9.0
```

Comment:
type="bool" option in "getini" method was added in pytest 2.9.0
We can use older pytest version and "ast" library for string->bool conversion